### ISSUE TITLE
All learning resources inside one tab

### ISSUE THREAD

**Issue by adityatiwatane** [stackup-dev/open-source-campaign#48]
**Aditya Tiwatane**
 #adityatiwatane

Is your feature request related to a problem? Please describe.
Sometimes we stackies need to learn from various resources for various quest or we have to practice for a specific technology like smart contract, Ethereum etc. To perfect our understanding about that technology. But we need to find that resource all the way from any quest of any campaign according to requirement for that we have to search for most of the campaigns and quest and only after that we get link of that particular resource. For example to learn  about solidity in solidity quest or ethereum I remembered about a platform where a proper material is available such as "Learneth" but I have to search it from previous campaigns and quest. As it is time consuming and frustrating to search for that link or resource.

**Describe the solution you'd like**
It is better if add a tab of "Resources" which includes all or important resources and links present in quests which can be referred easily as it is available according to its category such as blockchain named headline contains all links and resources related with blockchain in "Resources" section. 

**Describe alternatives you've considered**
Inside any campaign tab present a link containing "Resources" tab at top of that page.

**Additional context**
It makes accessing various content , links easier.

**Comment by vincenzo444** [comment]
yes, I would also like to recommend adding a resources page to easily navigate and fetch the required information rather than searching.
I would also recommend adding the best resources should be picked from the discord learning resources channel, instead of reading every resource link, we should probably keep the best and most useful in stackup website under different categories
